api_app/analyzers_manager/repo_downloader.sh

```diff
@@ -7,6 +7,9 @@
 # Example: you may want to add a new repository. Add the clone here
 # Example: you may want to remove some of the rules available in the downloaded repositories. Remove them here.
 
+# Install pre-bundled Onionscan (linux-amd64)
+chmod 755 /opt/deploy/bundled/onionscan
```

Comment:
why this line is in this file?

Reply:
Sir, I added that to make sure the bundled tools have correct permissions. Should I move it to the Dockerfile instead?

Reply:
yes, I think that line is unrelated to the that .sh file